vmdb/app/controllers/api_controller/events.rb

```diff
@@ -0,0 +1,12 @@
+class ApiController
+  module Events
+    #
+    # Events Subcollection Supporting Methods
+    #
+
+    def events_query_resource(object)
+      return {} unless object && object.respond_to?(:events)
```

Comment:
Can't this be simplified to `return {} unless object.respond_to?(:events)` because if `object` is `nil`, it will not `respond_to?(:events)`.